lib/main_containers.pm

```diff
@@ -13,7 +13,7 @@ use utils;
 use version_utils;
 use main_common qw(loadtest boot_hdd_image);
 use testapi qw(check_var get_required_var get_var set_var);
-use publiccloud::utils 'is_gce';
+use publiccloud::utils qw(is_azure is_gce);
```

Comment:
```suggestion
use publiccloud::utils qw(is_azure);
```

there is no other occurences of this function so it can be deleted from use section

Reply:
Done.